packages/webextension/index.js

```diff
@@ -0,0 +1,71 @@
+const web = require('@neutrinojs/web');
+const webExt = require('web-ext').default;
+const { join } = require('path');
+const merge = require('deepmerge');
+
+const MODULES = join(__dirname, 'node_modules');
+
+module.exports = (neutrino, opts = {}) => {
+  const handlePromiseRejection = err => {
+    console.error(err);
+    process.exit(1);
+  };
+
+  const options = merge({
+    webExtRun: {
+      noInput: true,
+      sourceDir: neutrino.options.output
+    },
+    minify: {}
+  }, opts);
+
+  neutrino.use(web, options);
+
+  if (neutrino.options.debug) {
+    webExt.util.logger.consoleStream.makeVerbose();
+  }
+
+  neutrino.config
+    .output
+      .delete('chunkFilename')
+      .hotUpdateChunkFilename('hot/hot-update.js')
+      .hotUpdateMainFilename('hot/hot-update.json')
+      .end()
+    .devServer
+      .clear()
+      .end()
+    .module
+      .rules
+        .delete('worker')
+        .end()
+      .end()
+    .plugins
+      .delete(options.pluginId || 'html')
+      .end()
+    .resolve
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .resolveLoader
+      .modules
+        .add(MODULES)
+        .end()
+      .end()
+    .when(neutrino.options.command === 'start', () => {
+      webExt.cmd
```

Comment:
I see where we are going with this, but exiting the process from here is going to skip all the error handling we have in the API.

What if we took this part out of this conditional and use an event?

```js
neutrino.on('start', () => webExt.cmd.run(options.webExtRun, { shouldExitProgram: false }));
```

This way would mean we could skip handling the promise altogether and let the API handle it. Give it a shot and let me know if that works.